engine/apps/api/urls/v1.py

```diff
@@ -109,5 +108,3 @@
     path(r"login/<backend>/", auth.overridden_login_slack_auth, name="slack-auth"),
     path(r"complete/<backend>/", auth.overridden_complete_slack_auth, name="complete-slack-auth"),
 ]
-
-urlpatterns += router.urls
```

Comment:
I'm fairly certain this is duplication of doing this:
```python3
urlpatterns = [
    path("", include(router.urls)),
   ...
]
```
![Screenshot 2023-07-14 at 11 10 22](https://github.com/grafana/oncall/assets/9406895/ed26b28d-9b2d-4f94-b84d-8a211e632d06)